### PR TITLE
TTAHUB-5224 - Collab Report Activity Methods filter

### PR DESCRIPTION
**Pull request by Andrew565**
## Description of change

CR Activity Methods filter

## How to test

1. Create or find a CR with a specific activity method
2. Filter by that activity method and ensure the filter works

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-5224

### Every PR

- [X] Meets issue criteria
- [X] JIRA ticket status updated
- [X] Code is meaningfully tested
- [X] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [X] API Documentation updated
- [X] Boundary diagram updated
- [X] Logical Data Model updated
- [X] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [X] UI review complete
- [ ] QA review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] PR created as **Draft**
- [ ] Staging smoke test completed
- [ ] PR transitioned to **Open**
- [ ] Reviewer added _(after transitioning to Open to ensure Slack notifications trigger; `elainaparrish` is the authorized approver under normal circumstances)_
  - _Sequence: Draft PR → Smoke test → Open PR → Add reviewer_
  - _Confirm that Slack notification was sent after reviewer was added_

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/scopes/collabReports/index.test.js [hunk 1]
```diff
@@ -35,4 +35,25 @@ describe('collabReports goal scope', () => {
 
     expect(scope).toEqual({});
   });
+
+  it('maps conductMethod.in to an IN where condition', () => {
```
Comment (kryswisnaskas) [inline src/scopes/collabReports/index.test.js:39]: Codex said:
Medium: the new coverage is too narrow to protect the real failure modes. The only added tests in index.test.js (line 39) assert the happy-path object shape; they do not cover empty input, invalid enum values, URL-parsed strings, or the service path that powers list/alerts/CSV. That leaves the main risk above completely unguarded, and future refactors could silently break the full filter pipeline while these tests still pass.

Concrete fix: add tests at the scope and service/handler levels. At minimum, cover conductMethod.in[]= and conductMethod.nin[]=bad, multiple valid values, and one request-level assertion that the collab reports endpoints ignore malformed values instead of erroring.

File: src/scopes/collabReports/conductMethod.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Op } from 'sequelize';
+
+export function withConductMethod(methods: string[]) {
+  return {
+    conductMethod: {
+      [Op.in]: methods,
```
Comment (kryswisnaskas) [inline src/scopes/collabReports/conductMethod.ts:20]: Codex said:
High: malformed conductMethod query values can take down the collab report list, alerts, and CSV endpoints. The new scope in conductMethod.ts (line 3) passes raw URL values straight into Op.in / Op.notIn on an enum column, and both getCSVReports (line 485) and getReports (line 581) consume that scope directly. A request like conductMethod.in[]= or conductMethod.nin[]=bogus is enough to produce an invalid enum comparison instead of being ignored, which turns a bad bookmark or hand-edited URL into a 500.

Concrete fix: normalize and validate before building the scope. Trim values, drop empties, keep only a whitelist like ['email', 'phone', 'in_person', 'virtual'], dedupe, and return {} when nothing valid remains. Add regression tests for invalid and empty conductMethod.in[] / conductMethod.nin[] inputs.

**Review by kryswisnaskas — APPROVED** [review]
Looks good! 👍
